### PR TITLE
Create email notif endpoint

### PR DESCRIPTION
**Pull request by cainsusk**
> NOTE: in **app.py** you must set
> 
> app.config['MAIL_PASSWORD'] = 'app password'
> 
> to the password sent in the discord (#resources)

creates an /email endpoint to send emails

check [the docs](http://localhost:5050/apidocs) to test

Screenshot 2024-03-28 at 9 38 07 PM

### CODE REVIEW

**Review by farahwamer — COMMENTED** [review]
Showing up on the docs! But when i try it out it says email sent, but i don't receive the email (i checked spam folders too).

**Comment by cainsusk** [comment]
OK it should work now @farahwamer just make sure to use the new app password (for some reason the old one was giving me trouble)

**Review by farahwamer — APPROVED** [review]
Works perfectlyyyyy!!!! just the env file and we good 🚀

File: app/app.py [hunk 1]
```diff
@@ -20,9 +23,47 @@
 api = Api(app)
 swagger = Swagger(app)
 
+mail= Mail(app)
+app.config['MAIL_SERVER']='smtp.gmail.com'
```
[the remark this answers is not in the document]
Comment (cainsusk) [reply app/app.py]: Yup !

**Comment by cainsusk** [comment]
@farahwamer I added the dotenv :)

**Comment by cainsusk** [comment]
and the .env file is in the discord

**Pull request merged**